core/arch/x86/decode.c

```diff
@@ -741,6 +741,7 @@ read_evex(byte *pc, decode_info_t *di, byte instr_byte,
             return pc;
         }
         *is_evex = true;
+        SYSLOG_INTERNAL_WARNING_ONCE(MSG_AVX_512_SUPPORT_INCOMPLETE_STRING " @" PFX, pc);
```

Comment:
If this were 2 months ago, I'd say let's use REPORT_FATAL_ERROR_AND_EXIT.  Should we still do that, with some temporary runtime option to avoid it for your development environment and Travis?  Or just have a printed message?

Reply:
I'd leave it as a syslog warning so we don't produce an artificial road block. And it's too late now, since it will soon be supported?

Reply:
OK.